Content.Server/Salvage/SalvageSystem.Magnet.cs

```diff
@@ -107,7 +107,7 @@ private void UpdateMagnet()
                 {
                     EndMagnet((uid, magnetData));
                 }
-                else if (!magnetData.Announced && (magnetData.EndTime.Value - curTime).TotalSeconds < 31)
+                else if (!magnetData.Announced && (magnetData.EndTime.Value - curTime).TotalSeconds < 59) //DeltaV: was 31 seconds. Increased to give time to actually fulton a crate out. 59 seconds picked because .Seconds only supports values up to 59.
```

Comment:
TimeSpan.Seconds does indeed only go up to 60, because it represents the seconds part of a TimeSpan
However, TimeSpan.TotalSeconds, which is what is used here, is what it says. A TimeSpan of 5 minutes and 1 second will have Seconds == 1 and TotalSeconds == 301
```suggestion
                else if (!magnetData.Announced && (magnetData.EndTime.Value - curTime).TotalSeconds < 59) //DeltaV: was 31 seconds. Increased to give time to actually fulton a crate out.
```